test/test.sh

```diff
@@ -23,6 +23,11 @@ else
         iface=lo
 fi
 
+if [ "$NSS_DIR" ] && [ "$NSS_TARGET" ]; then
```

Comment:
Huh, TIL.

 I would have used `[[ -n "$NSS_DIR" -a -n "$NSS_TARGET" ]]`.  I didn't even know that you could do `[ "$foo" ]`.  Is that even robust against the variable having a value of something like "!"?

Reply:
https://www.shellcheck.net/wiki/SC2108

Reply:
I love shellcheck

Reply:
`[` is `test`.

Reply:
Yeah, bash manual recommends `[[` over `[`/`test`.

The nice thing about shellcheck is that it is rarely wrong.  That's something to be treasured.